source/options.tsx

```diff
@@ -71,6 +72,25 @@ async function init(): Promise<void> {
 		container.prepend(unchecked.closest('.feature')!);
 	}
 
+	// Highlight new features
+	const newFeatures: Record<string, number> = await cache.get<Record<string, number>>('features-new') ?? {};
+	if (browser.runtime.getManifest().version !== '19.12.22') { // First version introducing highlighting
+		const timespan = 10 * 24 * 60 * 60 * 1000; // Highlighting will be shown for 10 days after opening the options
+		for (const feature of select.all('.feature [type=checkbox]')) {
+			if (!(feature.id in newFeatures) || Date.now() - newFeatures[feature.id] <= timespan) {
+				feature.parentElement!.classList.add('feature-new');
+			}
+		}
+	}
+
+	for (const feature of __featuresInfo__) {
+		if (!(feature.name in newFeatures)) {
+			newFeatures[feature.name] = Date.now();
+		}
+	}
+
+	await cache.set('features-new', newFeatures, 1000);
```

Comment:
I think you should just store them with `browser.storage.local.setItem`. 1000 days is almost 3 years

Reply:
`1000` was just an random high number, because it should never expire.

Reply:
Right, so it’s not cache. Cache is meant to expire